rst_files/getting_started.rst

```diff
@@ -206,16 +176,36 @@ If it's a long file, however, you have the alternative of
 The present working directory can be found by executing the command ``pwd()``
 
 
+Using QuantEcon Lecture Packages
+-------------------------------------------
+
+To use the curated set of packages in the QuantEcon lecture notes, 
+copy the following text into a notebook cell, and hit ``Shift-Enter`` to run the cell
+
+   .. literalinclude:: /_static/includes/deps.jl    
```

Comment:
@arnavs I would prefer to move this to the dependencies without includes literalinclude.  Arnav, can you help her out with that?

Reply:
We had already run this code earlier in the lecture I think...?

Reply:
I think you need to be very careful walking through the instructions berfore making radical changes.  Those instructions were done as part of the setup routine, not in a notebook.  These were all tested carefully, so we want to make sure we don't break the workflow.